examples/shapes_and_collections/fancybox_demo.py

```diff
@@ -15,17 +17,16 @@
 # First we'll show some sample boxes with fancybox.
 
 styles = mpatch.BoxStyle.get_styles()
-spacing = 1.2
-
-figheight = (spacing * len(styles) + .5)
-fig = plt.figure(figsize=(4 / 1.5, figheight / 1.5))
-fontsize = 0.3 * 72
-
-for i, stylename in enumerate(sorted(styles)):
-    fig.text(0.5, (spacing * (len(styles) - i) - 0.5) / figheight, stylename,
-             ha="center",
-             size=fontsize,
-             bbox=dict(boxstyle=stylename, fc="w", ec="k"))
+ncol = 2
+nrow = (len(styles) + 1) // ncol
+axs = plt.figure(figsize=(3 * ncol, nrow)).subplots(nrow, ncol)
+for ax in axs.flat:
+    ax.set_axis_off()
+for ax, (stylename, stylecls) in zip(axs.T.flat, styles.items()):
+    ax.text(.5, .5,
+            f"{stylename}\n{inspect.signature(stylecls)}",
```

Comment:
I'm not convinced this stylecls inspection is an improvement. This plot should primarily illustrate the available boxstyles. Adding the parameters is rather distracting. And also these are only the defaults, which should be irrelevant to most users (well maybe if you start modifying them, but that should be discussed separately.